src/internal/connector/data_collections.go

```diff
@@ -92,10 +92,7 @@ func (gc *GraphConnector) DataCollections(
 func verifyBackupInputs(sels selectors.Selector, userPNs, siteIDs []string) error {
 	var ids []string
 
-	resourceOwners, err := sels.ResourceOwners()
-	if err != nil {
-		return errors.Wrap(err, "invalid backup inputs")
-	}
+	resourceOwners := sels.DiscreteResourceOwners()
```

Comment:
how does this interact with passing "*" as the user? Or are we assuming that by this point "*" has been resolved to actual users?

Reply:
Right now we're still re-creating each selector within the per DiscreteScope loop.  The wildcard will cascade to a concrete value from that as expected.  Later on the wildcard will get resolved as part of selector creation/handling in a different way, but still prior to this point.